conf/defaults.ini

```diff
@@ -1063,6 +1063,9 @@ ha_redis_prefix =
 # provided, a random one will be generated.
 ha_redis_peer_name =
 
+# The maximum number of simultaneous redis connections.
+ha_redis_max_conns = 5
```

Comment:
I chose to call it `max_conns` instead of the redis library's `pool_size`.

I thought this was more readily apparent, and consistent with the same setting for databases. Happy to align with the redis library if anyone feels strongly, though.